### DISCUSSION TITLE
How to update druid DataSource

### DISCUSSION THREAD

**Discussion by rupeshpyeole** [Sunbird-Knowlg/Community#148] [General Discussions]
Hi everyone, we want to know which job runs to update "ml-survey-status" druid DataSource? 
Here we are facing issue is, we are not getting data in "ml-survey-status" bucket and hence data is not coming in druid DataSource. 
I, kindly request you to help us on this finding job for this druid data source or root cause of this issue. 

Thanks and regards, 
Rupesh

**Comment by pallakartheekreddy** [comment]
@rupeshpyeole Please raise this request in Ed https://github.com/orgs/Sunbird-Ed/discussions

**Reply by rupeshpyeole** [reply]
I appreciate your kind response ,Thank you so much.
